jniFaiss/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Design question: Should we have separate JNI libraries for FAISS and nmslib, or should they be contained in one?

Reply:
i am not sure which is better. when we make 2 libs as one, this may make jni code more compatible with different libs and different version but we server as one lib service. 
is there any possible to separate JNI libraries for FAISS and NMSLIB, but with one lib interface?

Reply:
I feel having seperate JNI would be more cleaner and easy to abstract out the underlying business logic to dedicated files.  I like the current approach.